<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreInterface</class>
 <widget class="QMainWindow" name="CoreInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>653</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TAT Main</string>
  </property>
  <property name="windowIcon">
   <iconset theme="qt">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="CoreLayout">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="EditorTabLayout" stretch="0">
      <item>
       <widget class="QTabWidget" name="EditorTabs">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="SettingsTab">
         <attribute name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/pics/resources/pics/emblems/emblem-system.png</normaloff>:/pics/resources/pics/emblems/emblem-system.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Settings</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QLabel" name="SubScoreLabel">
            <property name="text">
             <string>SubScore Items</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="HeaderEdit">
            <property name="placeholderText">
             <string>For example, the name of this exam.</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::VisualMoveStyle</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCommandLinkButton" name="StartButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="ClassSelectionMenu"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="ClassLabel">
            <property name="text">
             <string>Class</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="BeginLabel">
            <property name="text">
             <string>Begin</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="HeaderLabel">
            <property name="text">
             <string>Header</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="SubScoreItemsEdit">
            <property name="placeholderText">
             <string>For example, parts like &quot;vocabulary&quot;&quot;reading&quot;. Seperate them using comma(&quot;,&quot;), like &quot;grammar,vocabulary,writing&quot;</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::VisualMoveStyle</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="EditorTab">
         <attribute name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/pics/resources/pics/emblems/terminal.png</normaloff>:/pics/resources/pics/emblems/terminal.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Editor</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QLineEdit" name="CommandLine">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>commands here</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::VisualMoveStyle</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="ConfirmButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>10</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="text">
             <string>OK!</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextBrowser" name="CoreDisplay">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>255</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Processing">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="title">
          <string>Processing</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QGroupBox" name="SortLayout">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Sort</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <layout class="QVBoxLayout" name="SortSelectLayout">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetFixedSize</enum>
                 </property>
                 <item>
                  <layout class="QFormLayout" name="formLayout_2">
                   <property name="rowWrapPolicy">
                    <enum>QFormLayout::WrapAllRows</enum>
                   </property>
                   <property name="labelAlignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                   <property name="formAlignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                   </property>
                   <item row="0" column="0" colspan="2">
                    <widget class="QRadioButton" name="NumeralSortSelect">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Do nu&amp;meral sort</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLabel" name="NumeralSortExplainLabel">
                     <property name="font">
                      <font>
                       <pointsize>8</pointsize>
                       <italic>false</italic>
                       <underline>false</underline>
                       <strikeout>false</strikeout>
                       <kerning>true</kerning>
                      </font>
                     </property>
                     <property name="text">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#8b8b8b;&quot;&gt;If you're logging numbers&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout">
                   <item row="0" column="0">
                    <widget class="QRadioButton" name="DictionalSortSelect">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Do dictional sort</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="DictionalSortExplainLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>8</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#8b8b8b;&quot;&gt;If you're logging gradings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="sortButton">
                 <property name="text">
                  <string>Sort</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Preview</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QTextBrowser" name="previewDisplay"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="CoreMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
    <addaction name="AboutTATAction"/>
    <addaction name="ActionAboutQt"/>
    <addaction name="AboutTATActionKDE"/>
    <addaction name="AboutTATActionLegacy"/>
    <addaction name="TechnicalInfoAction"/>
   </widget>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Fi&amp;les</string>
    </property>
    <addaction name="ActionExit"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="AboutTATActionOld">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/resources/pics/Beacon.png</normaloff>:/pics/resources/pics/Beacon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About TAT(Legacy)</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="ActionAboutQt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/resources/pics/Qt.png</normaloff>:/pics/resources/pics/Qt.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="ActionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/resources/pics/emblems/emblem-unreadable.png</normaloff>:/pics/resources/pics/emblems/emblem-unreadable.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="AboutTATActionKDE">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/resources/pics/Beacon.png</normaloff>:/pics/resources/pics/Beacon.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;TAT (KDE KF5)</string>
   </property>
  </action>
  <action name="TechnicalInfoAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/resources/pics/emblems/statement.png</normaloff>:/pics/resources/pics/emblems/statement.png</iconset>
   </property>
   <property name="text">
    <string>Technical &amp;Info</string>
   </property>
  </action>
  <action name="AboutTATAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/resources/pics/Beacon.png</normaloff>:/pics/resources/pics/Beacon.png</iconset>
   </property>
   <property name="text">
    <string>A&amp;bout TAT</string>
   </property>
  </action>
  <action name="AboutTATActionLegacy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/resources/pics/Beacon.png</normaloff>:/pics/resources/pics/Beacon.png</iconset>
   </property>
   <property name="text">
    <string>About TAT (Legacy)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
